jwst/ramp_fitting/ramp_fit.py

```diff
@@ -336,14 +375,14 @@ def ols_ramp_fit(model, buffsize, save_opt, readnoise_model, gain_model,
 
     # Create new model...
     new_model = datamodels.ImageModel(data=c_rates.astype(np.float32),
-                                   dq=final_pixeldq.astype(np.int32),
-                                   err=err_cube[0, 0].copy())
+            dq=final_pixeldq.astype(np.int32), 
```

Comment:
Should dq be stored as uint32 (unsigned)? That's the way it's defined in the ImageModel schema.